public/components/query_compare/search_result/search_components/search_configs/search_configs.scss

```diff
@@ -6,6 +6,13 @@
 .search-relevance-config:nth-child(1) {
   margin: 0;
   padding: 12px;
-  border-right: 1px solid #D3DAE6;
+  border-right: 1px solid $euiBorderColor;
 }
-  
\ No newline at end of file
+
+.right-border {
+  border-right: 0;
+}
+
+.left-border {
+  border-left: 0;
+}
```

Comment:
Could this be in a single class?

Reply:
Yeah, I was actually thinking about putting them in one called "no-right-left-borders" but wasn't sure if there was a naming convention? If that works, I'll make the change.